src/Entity/Factures.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\FacturesRepository")
+ */
+class Factures
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="App\Entity\User", inversedBy="factures")
+     */
+    private $user;
+
+    /**
```

Comment:
Pas de lien entre RateCard et Factures

Reply:
CREATE TABLE rate_card (id INT AUTO_INCREMENT NOT NULL, facture_id INT DEFAULT NULL, solution VARCHAR(255) NOT NULL, prestation VARCHAR(255) NOT NULL, models VARCHAR(255) NOT NULL, price_rate_card INT NOT NULL, UNIQUE INDEX UNIQ_E9102ADC7F2DEE08 (facture_id), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB